greenlet.js

```diff
@@ -9,30 +9,30 @@ export default function greenlet(asyncFunction) {
 	// Outward-facing promises store their "controllers" (`[request, reject]`) here:
 	const promises = {};
 
-	// Create an "inline" worker (1:1 at definition time)
-	const worker = new Worker(
-		// Use a data URI for the worker's src. It inlines the target function and an RPC handler:
-		'data:,$$='+asyncFunction+';onmessage='+(e => {
-			/* global $$ */
+	// Use a data URI for the worker's src. It inlines the target function and an RPC handler:
+	const script = 'data:$$='+asyncFunction+';onmessage='+(e => {
+		/* global $$ */
 
-			// Invoking within then() captures exceptions in the supplied async function as rejections
-			Promise.resolve(e.data[1]).then(
-				v => $$.apply($$, v)
-			).then(
-				// success handler - callback(id, SUCCESS(0), result)
-				// if `d` is transferable transfer zero-copy
-				d => {
-					postMessage([e.data[0], 0, d], [d].filter(x => (
-						(x instanceof ArrayBuffer) ||
-						(x instanceof MessagePort) ||
-						(x instanceof ImageBitmap)
-					)));
-				},
-				// error handler - callback(id, ERROR(1), error)
-				er => { postMessage([e.data[0], 1, '' + er]); }
-			);
-		})
-	);
+		// Invoking within then() captures exceptions in the supplied async function as rejections
+		Promise.resolve(e.data[1]).then(
+			v => $$.apply($$, v)
+		).then(
+			// success handler - callback(id, SUCCESS(0), result)
+			// if `d` is transferable transfer zero-copy
+			d => {
+				postMessage([e.data[0], 0, d], [d].filter(x => (
+					(x instanceof ArrayBuffer) ||
+					(x instanceof MessagePort)
```

Comment:
We could add `self.ImageBitmap && x instanceof ImageBitmap` here to add support for transferring from the worker too. The `self` prefix is required in Safari for some reason.

Reply:
Oversight on my part, I went ahead and added that that in as recommended. It appears the code is building on Travis with the webpack devdep added. Let me know if there's anything more I can do